lib/SYCL/inSPIRation.cpp

```diff
@@ -44,6 +44,7 @@ STATISTIC(SYCLKernelProcessed, "Number of SYCL kernel functions processed");
 STATISTIC(SYCLFuncCalledInKernelFound, "Number of functions directly or indirectly called by SYCL kernel functions");
 
 cl::opt<bool> ReqdWorkGroupSizeOne("reqd-workgroup-size-1", cl::desc("set reqd_work_group_size to be 1-1-1"));
+cl::opt<bool> RemoveKernelArgType("remove-kernel-arg-type", cl::desc("remove kernel_arg_type"));
```

Comment:
Globally we do not really remove anything, we do not output it. :-)

What about renaming the options to:
```
 cl::opt<bool> ReqdWorkGroupSizeOne("spir-reqd-workgroup-size-1", cl::desc("set SPIR reqd_work_group_size metadata to be 1-1-1"));
cl::opt<bool> SkipKernelArgType("spir-skip-kernel-arg-type", cl::desc("do not output the SPIR  kernel_arg_type related metadata"));
```